top-sites/sites.js

```diff
@@ -0,0 +1,23 @@
+browser.topSites.get()
+  .then((sites) => {
+    var div = document.getElementById('site-list');
+
+    if (!sites.length) {
+      div.innerText = 'No sites returned from the topSites API.';
+      return;
+    }
+
+    let ul = document.createElement('ul');
+    ul.className = 'list-group';
+    for (let k in sites) {
```

Comment:
Is there a reason we're using for-in rather than for-of https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/?